### PR TITLE
[GraphBolt][Doc] update link prediction

### PR DESCRIPTION
**Pull request by Rhett-Ying**
## Checklist
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [$CATEGORY] (such as [NN], [Model], [Doc], [Feature]])
- [ ] I've leverage the [tools](https://docs.google.com/document/d/1iHyj7zlmygKSk5gBPsqIqL5ASPzJSPREaNT_QdsiYA4/edit) to beautify the python and c++ code.
- [ ] The PR is complete and small, read the [Google eng practice (CL equals to PR)](https://google.github.io/eng-practices/review/developer/small-cls.html) to understand more about small PR. In DGL, we consider PRs with less than 200 lines of core code change are small (example, test and documentation could be exempted).
- [ ] All changes have test coverage
- [ ] Code is well-documented
- [ ] To the best of my knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change
- [ ] Related issue is referred in this PR
- [ ] If the PR is for a new model/paper, I've updated the example index [here](../examples/README.md).

### CODE REVIEW

File: tutorials/large/L2_large_link_prediction.py [hunk 1]
```diff
@@ -23,62 +23,53 @@
...
 os.environ["DGLBACKEND"] = "pytorch"
-import dgl
+import dgl.graphbolt as gb
 import numpy as np
 import torch
-from ogb.nodeproppred import DglNodePropPredDataset
+import tqdm
 
-dataset = DglNodePropPredDataset("ogbn-arxiv")
+dataset = gb.BuiltinDataset("ogbl-citation2").load()
```
Comment (frozenbugs) [inline tutorials/large/L2_large_link_prediction.py:53]: nit: Will this be very slow if we use ogbl-citation2?

File: tutorials/large/L2_large_link_prediction.py [hunk 2]
```diff
@@ -94,367 +85,249 @@
...
-# storing the computation dependencies for each GNN layer.
-# The MFGs are used to compute the GNN outputs of the nodes
-# involved in positive/negative graph.
-#
 
+import dgl.nn as dglnn
 
 ######################################################################
-# Defining Model for Node Representation
+# Defining Model for Link Prediction
```
Comment (frozenbugs) [inline tutorials/large/L2_large_link_prediction.py:118]: Node Representation is actually accurate.

**Review by frozenbugs — APPROVED** [review]

File: tutorials/large/L2_large_link_prediction.py (hunk 1 above)
Comment (Rhett-Ying) [reply tutorials/large/L2_large_link_prediction.py:53]: Yes, it takes about 5 min. It's a bit unacceptable

Comment (Rhett-Ying) [reply tutorials/large/L2_large_link_prediction.py]: replaced with `cora` now

**Pull request merged**